pkg/domain/login/auth_test.go

```diff
@@ -2,59 +2,50 @@ package login
 
 import (
 	"context"
-	"errors"
 	"testing"
 	"time"
 
 	accounts_usecases "github.com/daniel1sender/Desafio-API/pkg/domain/accounts"
 	"github.com/daniel1sender/Desafio-API/pkg/domain/entities"
 	"github.com/daniel1sender/Desafio-API/pkg/gateways/store/postgres/accounts"
+	"github.com/daniel1sender/Desafio-API/pkg/tests"
 	"github.com/golang-jwt/jwt/v4"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestLoginUseCase_Auth(t *testing.T) {
 	ctx := context.Background()
 	accountRepository := accounts.NewStorage(Db)
 	tokenSecret := "123"
 	useCase := LoginUseCase{accountRepository, tokenSecret}
+	assert := assert.New(t)
+
 	t.Run("should return a signed token", func(t *testing.T) {
 		name := "Jonh Doe"
 		cpf := "01481623559"
 		accountSecret := "123"
 		balance := 10
-		account, err := entities.NewAccount(name, cpf, accountSecret, balance)
-		if err != nil {
-			t.Errorf("expected no error while creating a new account but got '%s'", err)
-		}
+		account, _ := entities.NewAccount(name, cpf, accountSecret, balance)
 		duration := "1m"
 		accountRepository.Upsert(ctx, account)
 		tokenString, err := useCase.Auth(ctx, account.CPF, accountSecret, duration)
-		if err != nil {
-			t.Errorf("expected no error but got '%v'", err)
-		}
-		if len(tokenString) == 0 {
-			t.Error("got empty token")
-		}
+		assert.NoError(err)
+		assert.NotEmpty(tokenString, "got empty token")
```

Comment:
O NotEmpty passa dois parâmetros: o test e a variável que você quer testar se é vazia. Portanto ficaria algo tipo:
```suggestion
		assert.NotEmpty(t, tokenString)
```